app/src/main/java/com/example/alpaki/presentation/main/MainActivity.kt

```diff
@@ -21,11 +24,25 @@ class MainActivity : AppCompatActivity() {
     @Inject
     lateinit var navigationHandler: NavigationHandler
 
+    @Inject
+    lateinit var exampleUseCase: ExampleUseCase
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-        exampleInjectedClass.doSomething()
-        injectedActionBar.height
         navigationHandler.initNavigation(this)
+
+        exampleUseCase(lifecycleScope, None, { result ->
```

Comment:
rozumiem, że jak zrobimy viewModele to będziemy useCase'y tam używać albo spięte w jakimś repository/service?

Reply:
Tak jest, UseCase do ViewModeli i one będą już gadały z repo etc.